Multisig/UI/App/HeaderViewController/HeaderViewController.swift

```diff
@@ -38,6 +40,11 @@ final class HeaderViewController: ContainerViewController {
         addObservers()
         headerBarHeightConstraint.constant = ScreenMetrics.safeHeaderHeight
         reloadSafeData()
+
+        #warning("Enable ribbon after Safe model implementation")
+        ribbonView.text = "Rinkeby"
```

Comment:
I have two ideas here, 
1) we need to make this view as component and observe the change of network info, I already implemented post notification when changing the network so instead of doing this in all view controllers how about just do this in the ribbonView?

2) can't we find a way to add this ribbon in all view controllers together instead of adding it in the header in some and at top of others?

Reply:
this would be the next step when implementing it in other places, for now I kept it here because there's also a shadow that needs to be on top of the content view and ribbon is above the shadow